src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,82 @@
+@use "../../variables" as *;
+.footer {
+  font-size: $fontsize-s;
+  .contact {
+    text-align: center;
+    padding: 32px 0;
+    address {
+      line-height: 28px;
+      margin: 20px;
+    }
+    ul {
+      display: flex;
+      justify-content: center;
+      margin: 0;
+      padding: 0;
+      li {
+        list-style-type: none;
+        i {
+          color: $black;
+          margin: 0 7px;
+          transition: ease 0.2s;
+          &:hover {
+            color: $gray_nobel;
+          }
+        }
+      }
+    }
+  }
+  nav {
+    ul {
+      border-top: solid 1px $gray_nobel;
+      border-bottom: solid 1px $gray_nobel;
+      background-color: $white_smoke;
+      display: grid;
+      grid-template-columns: repeat(2, 1fr);
+      margin: 0;
+      padding: 12px 16px;
```

```suggestion
      padding: 16px;
```
gapを設定すれば縦横16で大丈夫そうです！